dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloader.java

```diff
@@ -53,7 +59,32 @@
     private final ProgressBar progressBar = ProgressBar.getInstance();
```

Comment:
I know these aren't your changes here but I would prefer these variables to be set within a constructor. We could still have a 0-argument constructor